packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -439,6 +439,19 @@ export default function useAutocomplete(props) {
       const item = newValue;
       newValue = Array.isArray(value) ? [...value] : [];
 
+      if (process.env.NODE_ENV !== 'production') {
+        const matches = newValue.filter(val => getOptionSelected(item, val));
+
+        if (matches.length > 1) {
+          console.error(
+            [
+              'Material-UI: the `getOptionSelected` method of useAutocomplete do not handle the arguments correctly.',
+              `The component expects a single value to match a given option but found ${matches}.`,
```

Comment:
I wouldn't mention what used this method. Just that this function should only match a single option.

Reply:
oops i'm sorry turns out it logs
`... but found [object Object], [object Object], [object Object]`,

so i'll just delete those i think

Reply:
matches should be a number then?

Reply:
how about `... but found ${matches.length} matches` ?